app/code/Magento/Bundle/Pricing/Price/ConfiguredPrice.php

```diff
@@ -48,6 +48,10 @@ class ConfiguredPrice extends CatalogPrice\FinalPrice implements ConfiguredPrice
      * @var ConfiguredPriceSelection
      */
     private $configuredPriceSelection;
+    /**
```

Comment:
You have missed adding an empty line above the property declaration.
```suggestion

    /**
```

Reply:
I have add the missing line but github not see it